com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/PBRForwardPass.hlsl

```diff
@@ -72,14 +86,15 @@ half4 frag(PackedVaryings packedInput) : SV_TARGET
         float metallic = surfaceDescription.Metallic;
     #endif
 
-    SurfaceData surface         = (SurfaceData)0;
+    SurfaceData surface;
```

Comment:
I feel the opposite here - the compiler warns here when we don't initialize these elements and we should be trying to eliminate all compiler warnings. One for @phi-lira to decide on

Reply:
I should have been through an changed all of these back to initialization like above - I don't think it's a nice solution, the "Create...()" approach is far cleaner since it doesn't rely on "out" parameters and hence uninitialized structs being declared in the code BUT that's not how everything is currently implemented, so I've made may changes match the existing approach.